### DISCUSSION TITLE
Can I use the HTML action attribute

### DISCUSSION THREAD

**Discussion by transcental** [react-hook-form/react-hook-form#8801] [Q&A]
Hi,
I have a form with a file input as well as a few text inputs. I then make a POST request in my `onSubmit` method to a backend API endpoint written in Python's Flask framework.

I'm able to access all of the data via the `request.data` method. Apart from the file which is returned as:
`{'file': {'0': {}}}`
However, I believe that if I sent it as a normal form (via the `action` attribute) I'd be able to access the file data.

So, my question is whether it's possible to use the `action` attribute with `react-hook-form` instead of `onSubmit`

**Comment by bluebill1049** [comment]
It's ok to native form without react hook form and you can leverage `action` and post to your web server.
